cdap-ui/app/hydrator/templates/detail/top-panel.html

```diff
@@ -58,42 +58,42 @@
           <div ng-click="TopPanelCtrl.do('Start')"
                class="btn"
-               ng-class="{'btn-select': TopPanelCtrl.runPlayer.action === 'STARTING'}"
-               ng-disabled="TopPanelCtrl.appStatus === 'STARTING'">
-            <span ng-if="TopPanelCtrl.appStatus !== 'STARTING'">
+               ng-class="{'btn-select': TopPanelCtrl.runPlayer.action === 'Starting'}"
+               ng-disabled="TopPanelCtrl.appStatus === 'Starting'">
+            <span ng-if="TopPanelCtrl.appStatus !== 'Starting'">
               <span class="fa fa-play text-success"></span>
               <div class="button-label">Start</div>
             </span>
-            <span ng-if="TopPanelCtrl.appStatus === 'STARTING'">
+            <span ng-if="TopPanelCtrl.appStatus === 'Starting'">
               <span class="fa fa-spinner fa-spin"></span>
             </span>
           </div>
         </div>
-        <div ng-if="['STOPPING', 'RUNNING'].indexOf(TopPanelCtrl.appStatus) !== -1">
+        <div ng-if="['Stopping', 'Running'].indexOf(TopPanelCtrl.appStatus) !== -1">
           <div ng-click="TopPanelCtrl.do('Stop')"
                class="btn"
-               ng-disabled="TopPanelCtrl.appStatus === 'STOPPING'">
-            <span ng-if="TopPanelCtrl.appStatus !== 'STOPPING'">
+               ng-disabled="TopPanelCtrl.appStatus === 'Stopping'">
+            <span ng-if="TopPanelCtrl.appStatus !== 'Stopping'">
               <span class="fa fa-stop text-danger"></span>
               <div class="button-label">Stop</div>
             </span>
-            <span ng-if="TopPanelCtrl.appStatus === 'STOPPING'">
+            <span ng-if="TopPanelCtrl.appStatus === 'Stopping'">
               <span class="fa fa-spinner fa-spin"></span>
             </span>
           </div>
         </div>
       </div>
       <div ng-if="TopPanelCtrl.GLOBALS.etlBatchPipelines.indexOf(TopPanelCtrl.app.type) !== -1">
-        <div ng-if="['SUSPENDED', 'SCHEDULING'].indexOf(TopPanelCtrl.scheduleStatus) !== -1 || TopPanelCtrl.scheduleStatus.error">
+        <div ng-if="['Deployed', 'Scheduling'].indexOf(TopPanelCtrl.scheduleStatus) !== -1 || TopPanelCtrl.scheduleStatus.error">
 
           <div ng-click="TopPanelCtrl.do('Schedule')"
               class="btn scheduler"
-              ng-class="{'btn-select': TopPanelCtrl.runPlayer.action === 'SCHEDULING'}"
-            ng-disabled="TopPanelCtrl.scheduleStatus === 'SCHEDULING' || TopPanelCtrl.scheduleStatus.error">
-            <span class="double-line" ng-if="(TopPanelCtrl.scheduleStatus === 'SUSPENDED' || TopPanelCtrl.scheduleStatus.error) && !TopPanelCtrl.scheduleLoading">
+              ng-class="{'btn-select': TopPanelCtrl.runPlayer.action === 'Scheduling'}"
+            ng-disabled="TopPanelCtrl.scheduleStatus === 'Scheduling' || TopPanelCtrl.scheduleStatus.error">
+            <span class="double-line" ng-if="(TopPanelCtrl.scheduleStatus === 'Deployed' || TopPanelCtrl.scheduleStatus.error) && !TopPanelCtrl.scheduleLoading">
```

Comment:
This and line 96 seem different. Can the scheduleStatus be 'Deployed' or 'Suspending'? I thought we always mapped 'SUSPENDED' to 'Deployed'